recipes/mold/1.3.1/conanfile.py

```diff
@@ -82,11 +83,9 @@ def build(self):
             autotools.make(target="mold", args=['SYSTEM_TBB=1', 'SYSTEM_MIMALLOC=1'])
 
     def package(self):
-        self.copy("LICENSE", src=self._source_subfolder, dst="licenses")
-        self.copy("mold", src=self._source_subfolder, dst="bin", keep_path=False)
-
-    def package_id(self):
-        del self.info.settings.compiler
```

Comment:
Why did you remove this from `package_id`?

Reply:
Because I've got an error in the CI:

```
[HOOK - conan-center.py] post_package(): ERROR: [MATCHING CONFIGURATION (KB-H014)] Packaged artifacts does not match the settings used: os=Linux, compiler=gcc (https://github.com/conan-io/conan-center-index/blob/master/docs/error_knowledge_base.md#KB-H014) 
```

Check build 16

Reply:
We need to keep this. Otherwise using these packages as tool_requirements becomes annoying
https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings

Reply:
Revert. We need to find a fix for it. Compiler should not be part of the package ID. It's documented here: https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#settings